cypress-io/cypress
# chore: adding element selectors to highlight in snapshots

### Additional details

* Uses the `@cypress/unique-selector` package to return the selectors for the `$elToHighlight` element(s).

### Steps to test


- Checkout mschile/protocol/highlight_element branch from the services repo
- Record a spec that `cy.get`'s an element
- Verify the command log event in the db includes the `elToHighlightSelectors` array with the selector for the element.

### How has the user experience changed?

`elToHighlightSelectors` is now included in the payload:

```json
{
  "id": "log-https://example.cypress.io-4",
  "name": "get",
  "message": ".as-table",
  "type": "parent",
  "testCurrentRetry": 0,
  "state": "passed",
  "testId": "r4",
  "wallClockUpdatedAt": "2023-05-25T21:27:58.954Z",
  "snapshots": [
    {
      "timestamp": 1685050078953.7998, 
      "elToHighlightSelectors": [".as-table"]
    }
  ],
  "event": false,
  "numElements": 1,
  "timeout": 4000,
  "visible": true,
  "wallClockStartedAt": "2023-05-25T21:27:58.948Z",
  "hookId": "r4",
  "instrument": "command"
}
```

### PR Tasks
